agent/Godeps/Godeps.json

```diff
@@ -1,12 +1,14 @@
 {
 	"ImportPath": "github.com/aws/amazon-ecs-agent/agent",
 	"GoVersion": "go1.6",
-	"GodepVersion": "v60",
-	"Packages": [
-		"./..."
```

Comment:
Looks like you might have only saved one package instead of all the packages used by the agent.